fun.txt

```diff
@@ -0,0 +1 @@
+this is a joke
```

Comment:
No it is not.

Reply:
Hey ,
I forgot to delete this PR I made this while giving a session to show how to make an PR 😅🤣

Reply:
So do your tests in your own repos.